WalletWasabi.Tests/AcceptanceTests/HwiKatas.cs

```diff
@@ -99,6 +99,47 @@ public async Task TrezorTKataAsync()
+
+			// USER: CONFIRM
+			PSBT signedPsbt = await client.SignTxAsync(deviceType, devicePath, psbt, cts.Token);
+
+			Transaction signedTx = signedPsbt.GetOriginalTransaction();
+			Assert.Equal(psbt.GetOriginalTransaction().GetHash(), signedTx.GetHash());
+
+			var checkResult = signedTx.Check();
```

Comment:
Why do you need this variable? Was it for debugging?

Reply:
I really don't know, I just modified the test that @lontivero wrote